src/app/components/scripts/scripts.menus.ts

```diff
@@ -4,58 +4,54 @@ import {
   defaultEditIcon,
   defaultNewIcon,
   isAdminPredicate,
+  isLoggedInPredicate,
 } from "src/app/app.menus";
 import { CommonRouteTitles } from "src/app/stringConstants";
-import { adminDashboardMenuItem, adminRoute } from "../admin.menus";
-
-export const adminScriptsRoute = adminRoute.addFeatureModule("scripts");
+import { scriptRoute, scriptsRoute } from "./scripts.routes";
 
 export const adminScriptsCategory: Category = {
   icon: ["fas", "scroll"],
   label: "Scripts",
-  route: adminScriptsRoute,
+  route: scriptsRoute,
 };
 
-export const adminScriptsMenuItem = menuRoute({
+export const scriptsMenuItem = menuRoute({
   icon: ["fas", "scroll"],
   label: "Scripts",
   route: adminScriptsCategory.route,
   tooltip: () => "Manage custom scripts",
-  parent: adminDashboardMenuItem,
-  predicate: isAdminPredicate,
-});
-
-export const adminNewScriptsMenuItem = menuRoute({
-  icon: defaultNewIcon,
-  label: "New Script",
-  route: adminScriptsMenuItem.route.add("new"),
-  tooltip: () => "Create a new script",
-  parent: adminScriptsMenuItem,
-  predicate: isAdminPredicate,
+  predicate: isLoggedInPredicate,
 });
 
-const adminScriptRoute = adminScriptsRoute.add(":scriptId");
-
-export const adminScriptMenuItem = menuRoute({
+export const scriptMenuItem = menuRoute({
   icon: ["fas", "scroll"],
   label: "Script",
-  route: adminScriptRoute,
+  route: scriptRoute,
   tooltip: () => "Manage script",
-  parent: adminScriptsMenuItem,
-  predicate: isAdminPredicate,
+  predicate: isLoggedInPredicate,
+  parent: scriptsMenuItem,
   title: (routeData: RouterStateSnapshot): string => {
     const componentModel = routeData.root.firstChild.data;
     const scriptName = componentModel.script.model.name;
     return scriptName;
   }
 });
 
+export const newScriptMenuItem = menuRoute({
+  icon: defaultNewIcon,
+  label: "New Script",
+  route: scriptsMenuItem.route.add("new"),
```

Comment:
is this going anywhere?

Reply:
You can access this route if you are an admin by navigating through

Admin dashboard > Scripts > New Script (in side bar)

Reply:
yeah but we don't have UI for it right?

Reply:
We do and it was automatically updated when the `scripts.schema` was updated

https://github.com/QutEcoacoustics/workbench-client/blob/master/src/app/components/admin/scripts/new/new.component.ts

It doesn't have any of the fancy syntax highlighting that we want to do in the future, but it exists right now